python/sdss_access/sync/auth.py

```diff
@@ -21,11 +22,17 @@ def __init__(self, netloc=None, public=False, verbose=False):
     def set_netrc(self):
         try:
             from netrc import netrc
-            self.netrc = netrc() if not self.public else None
         except Exception as e:
+            netrc = None
             if self.verbose:
                 print("SDSS_ACCESS> AUTH NETRC: %r" % e)
-            self.netrc = None
+        if netrc:
+            """ Windows: recommending _netrc following
```

Comment:
this comment should be moved into the docstring for `set_netrc` at line 23, with some expanded text on what `set_netrc` does.  It will help with readability.

Reply:
that's fine with me.